particles.py

```diff
@@ -0,0 +1,149 @@
+"""
+particles.py
+A python package for the MolSSI Software Summer School.
+Contains Particles class and subclasses RandomParticles and FetchParticles
+"""
+
+import numpy as np
+from box import *
+from abc import ABC, abstractmethod
+
+class Particles(ABC):
```

Comment:
If you really want to make `Particles` the only main class, then you should look into the `classmethod`
decorator. Then you can have `classmethod`s `from_random` and `from_file` to generate particles as 
constructors for the Particle class. Then you wouldn't need a `_generate_coordinates` method.

Then you would only need one `Particles` class.

Reply:
Thanks! Sounds interesting... I will work on it!